src/client/components/DetailsModal/index.tsx

```diff
@@ -211,39 +207,49 @@ export const DetailsModal = ({
     startDate,
     data: {
       ...mainQuoteData,
-      isStudent: quoteSelector.isStudent(mainQuote),
-      householdSize: numberCoInsured + 1,
+      ...(!quoteSelector.isCar(mainQuote) && {
+        isStudent: quoteSelector.isStudent(mainQuote),
+        householdSize: numberCoInsured + 1,
+      }),
     },
   } as QuoteInput
 
   const isUnderwritingGuidelinesHit = isLimitHit(createQuoteBundleData)
   const isQuoteCreationFailed = isQuoteBundleError(createQuoteBundleData)
 
   const reCreateQuoteBundle = (form: QuoteInput) => {
-    const {
-      data: { householdSize },
-    } = form
-
+    const quotes = allQuotes.map((quote) => {
+      const { startDate, currentInsurer, data } = quote
+      const {
+        data: { zipCode, street, city },
+      } = form
+      return {
+        ...form,
+        startDate,
+        currentInsurer: currentInsurer?.id,
+        data: {
+          ...(quoteSelector.isCar(mainQuote) === quoteSelector.isCar(quote)
```

Comment:
I guess because of the nested nature of the data, that it's difficult to automatically get the properties of the form data based on what `quote` looks like?

What I mean is that, if `quote` looks like this:

```
{
    registrationNumber: 'abc123'
}
```
then we would know that we would like to get key `registrationNumber` from `form.data`. Then we wouldn't need any `if` statements for Car in this case. But perhaps that's not possible?